### PR TITLE
feat(generators/component): Changed the functions to new Flynt setup

### PR DESCRIPTION
**Pull request by roelti**
@domtra I made this PR so the generator uses the correct functions for the new Flynt setup

### CODE REVIEW

File: generators/component/templates/functions.php [hunk 1]
```diff
@@ -2,49 +2,27 @@
...
-                'name' => 'options',
-                'type' => 'group',
-                'layout' => 'row',
-                'sub_fields' => [
-                    Api::loadFields('FieldVariables', 'theme'),
-                ],
-            ],
-        ],
-    ],
-]);
```
Comment (domtra) [inline generators/component/templates/functions.php:42]: why did you delete this? it does show our general approach of having a general and an options tab, and how to use the FieldVariables. The last point would have to be adjusted to the the syntax, though.

Comment (roelti) [reply generators/component/templates/functions.php]: @domtra I thought that the VScode snippets were meant for that and keeping it empty would save some time deleting the "standard" text but I see your point and I agree. I have made a couple adjustments and pushed to the branch.

File: generators/component/templates/functions.php [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
 
-Api::registerFields('<%= nameUpperCamelCase %>', [
-    'layout' => [
-        'name' => '<%= nameLowerCamelCase %>',
-        'label' => '<%= namePrettySplit %>',
+function getACFLayout()
+{
+    return [
+        'name' => '<%= nameUpperCamelCase %>',
+        'label' => '<%= namePretty %>',
```
Comment (domtra) [inline generators/component/templates/functions.php:16]: why were these two changed? the name should be lowerCamelCase, no?
also the label is now wrapped in a `__()`. could you include that, please?

Comment (roelti) [reply generators/component/templates/functions.php]: Correct! I fixed the issues

File: generators/component/templates/functions.php [hunk 3]
```diff
@@ -2,27 +2,28 @@
...
     return $data;
 });
 
-Api::registerFields('<%= nameUpperCamelCase %>', [
-    'layout' => [
+function getACFLayout()
+{
+    return [
         'name' => '<%= nameLowerCamelCase %>',
         'label' => '<%= namePrettySplit %>',
```
Comment (domtra) [inline generators/component/templates/functions.php:16]: sorry, but this would also need the `__()`

Comment (roelti) [reply generators/component/templates/functions.php]: ah right. Fixed now!
